### ISSUE TITLE
Front end won't update even if the shuttle's location changed in backend

### ISSUE THREAD

**Issue by KeyboardNerd** [KeyboardNerd/shuttle_tracking_2#29]
Maybe the synchronization between backend and front end should be improved, it's sometimes weird that the backend says that a shuttle arrives at a stop but the front end is the old data...

**Comment by connection-lost** [comment]
The front end always check in 15 second interval, while back end has a higher check interval.

Here are what back end (@KeyboardNerd) should probably do:
1. Find out where back end update rate is and how to change it.
2. Make it configurable in the configuration. This allows future changes. (What if iTrak changes their policy).
3. iTrak currently updates every 15 seconds. It would be nice that we always update our information immediately after iTrak has been updated.
4. Probably make a timestamp variable that sends to front end for debug purpose. It is currently impossible to tell when the information is generated in the front end.

**Comment by KeyboardNerd** [comment]
I agree with that. Maybe it should be listed as another milestone...
